doc/start/quick-ceph-deploy.rst

```diff
@@ -68,9 +68,9 @@ configuration details, perform the following steps using ``ceph-deploy``.
 
      [Ubuntu 18.04] $ sudo apt install python-minimal
 
-#. If you have more than one network interface, add the ``public network``
-   setting under the ``[global]`` section of your Ceph configuration file.
-   See the `Network Configuration Reference`_ for details. ::
+#. Add the ``public network`` setting under the ``[global]`` section of your
+   Ceph configuration file. See the `Network Configuration Reference`_ for
+   details. ::
 
```

Comment:
There is no need to add a network config if you you only have 1 interface.
Even if you have two interfaces, bonding is recommended.
third paragraf of: https://docs.ceph.com/docs/master/rados/configuration/network-config-ref/
I run all my clusters without any network setting, ceph is smart enough by itself

Reply:
When following the quick start instructions, the problem I've run into is that additional monitor nodes can't be added to the cluster if `public network` is not defined.

The error given is:

> [**WARNIN**] neither `public_addr` nor `public_network` keys are defined for monitors

Does `public network` need to be defined in order to add additional monitors? Or is there something else missing from the quick start that would prevent this error?

Reply:
@RedXanadu i think what is missing might be a link to https://docs.ceph.com/docs/master/rados/deployment/ceph-deploy-mon/#add-a-monitor. but i think it's fine. this is a "quick" ceph-deploy guide, and if one wants to add an additional monitor, https://docs.ceph.com/docs/master/rados/deployment/ceph-deploy-mon/#add-a-monitor is supposed to offer this additional help =)

Reply:
@tchaikov Apologies for the delay; this fell to the side for a while.
I agree with your comments. I see now that the “Add a Monitor” page fills in the gaps.
Is there a simple way to withdraw this single commit? Also I think there are merge conflicts now. Does this need to be a new pull request?

Reply:
I've reverted this commit, but I _suspect_ a new pull request is required now due to merge conflicts since the new documentation for Octopus was pushed out recently

Reply:
This commit was reverted after discussion. Marking conversation as resolved